### ISSUE TITLE
Optimize with CV and specific scoring

### ISSUE THREAD

**Issue by psinger** [zygmuntz/hyperband#5]
Hi! I am wondering whether it is possible to optimize with cross validation and preferably with a custom scoring function. Currently, it picks the configuration that minimizes e.g., the log loss of the training data if I am not mistaken. Would be good to also have similar options as grid search offers in scikit learn.

**Comment by zygmuntz** [comment]
Hi! It is straightforward to use another loss function. For example, when using a scikit-learn compatible classifier, see this:
https://github.com/zygmuntz/hyperband/blob/master/common_defs.py#L63

You could also modify that very function to use cross-validation.

**Comment by psinger** [comment]
Thanks, already figured it out :)
